packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -1002,4 +1023,11 @@ export class FocusZone extends React.Component<IFocusZoneProps, {}> implements I
   private _shouldWrapFocus(element: HTMLElement, noWrapDataAttribute: 'data-no-vertical-wrap' | 'data-no-horizontal-wrap'): boolean {
     return !!this.props.checkForNoWrap ? shouldWrapFocus(element, noWrapDataAttribute) : true;
   }
+
+  /**
+   * Returns true if the element is a descendant of the FocusZone through a React portal.
+   */
+  private _isDescendantThroughPortal(element: HTMLElement): boolean {
+    return element && portalContainsElement(element, this._root.current ? this._root.current : undefined);
```

Comment:
Is `this._root.current` being undefined a valid case here? I'm curious if we still want to call `portalContainsElement` if it is actually undefined...

Reply:
Yes. That is wrong. When it is null or undefined, we should return false directly without calling portalContainsElement. Correcting that.